src/main/java/quiz/model/QuizRound.java

```diff
@@ -59,11 +60,53 @@ public boolean isComplete() {
 		return region != null && players != null && players.length > 0;
 	}
 	
-	public void createPlayers(int count) {
+	public void createPlayers(int count, StringBuilder speechText) {
 		// TODO: Range validation
 		players = new Player[count];
-		for (int i=0; i<count; i++)
-			players[i] = new Player("Spieler "+(i+1), 0);
+			if(count == 1) {
+				players[0] = new Player("",0);
+				speechText.append("Alles klar. Dann spielen nur wir beide! Los geht´s mit deinen ersten fünf Fragen.");
+			}
+			else if(count == 2) { 
+				players[0] = new Player("Heidi",0);
+				players[1] = new Player("Peter",0);
+				speechText.append("Spitze. Zu zweit macht's immer mehr Spaß."
+						+ " Ich nenn euch jetzt einfach mal Heidi und Peter."
+						+ " Ich stelle euch jeweils abwechselnd 5 Fragen."
+						+ " Ladies first! Auf geht's, Heidi!");

```

Comment:
Das funktioniert so nicht, wenn man erst die Spieleranzahl eingibt und erst danach die Region. "Auf gehts" (und ähnliche Äußerungen in den anderen Fällen) sind dann fehl am Platze. Das bitte erst (und nur) bei `if (isComplete) askQuestion()` ausgeben.